dvc/tree/gdrive.py

```diff
@@ -128,16 +118,17 @@ def __init__(self, repo, config):
         self._client_id = config.get("gdrive_client_id")
         self._client_secret = config.get("gdrive_client_secret")
         self._validate_config()
-        self._gdrive_user_credentials_path = (
-            tmp_fname(os.path.join(self.repo.tmp_dir, ""))
-            if os.getenv(GDriveTree.GDRIVE_CREDENTIALS_DATA)
-            else config.get(
-                "gdrive_user_credentials_file",
-                os.path.join(
-                    self.repo.tmp_dir, self.DEFAULT_USER_CREDENTIALS_FILE,
-                ),
+
+        if os.getenv(GDriveTree.GDRIVE_CREDENTIALS_DATA):
```

Comment:
@julsal is it a purely cosmetic change? could we please, keep changes like this for a separate PR? Two reasons- they are subjective and might create some discussion that are not relevant, they extremely slow down the review process since you have to spend some time looking into them to see if there is some relevant logic changes. Let's keep the for the next, housekeeping iteration please. Let's try to keep only relevant changes in this (non small already PR). Thanks 🙏

Reply:
Sure! I found it difficult to understand, so I ended up changing. But, I'll make the PR free of cosmetics.